packaging/aznfs/DEBIAN/control

```diff
@@ -0,0 +1,11 @@
+Package: aznfs
+Version: x.y.z
+Priority: required
+Architecture: amd64
+Origin: Microsoft
+Maintainer: Azure Storage XNFSv3 IDC Team <xnfsv3devsidc@microsoft.com>
+Homepage: https://github.com/Azure/BlobNFS-mount/blob/main/README.md
+Description: Mount helper program to keep NFSv3 mount points working after account migration.
```

Comment:
this is public facing, so needs to be intentionally subtle, just say this one line.

Mount helper program for correctly handling endpoint IP address changes for Azure Blob NFS mounts.